homeassistant/helpers/template.py

```diff
@@ -1380,6 +1381,18 @@ def state_attr(hass: HomeAssistant, entity_id: str, name: str) -> Any:
     return None
 
 
+def is_available(hass: HomeAssistant, entity_id: str | Iterable) -> bool:
```

Comment:
The naming is kinda misleading, as it does not just check for availability but also the unknown state. Something unknown doesn't mean it is unavailable.

Reply:
Someone suggested is_nominal within the forum. Would that work?